data/net/minecraft/data/models/BlockModelGenerators.mapping

```diff
@@ -326,6 +326,8 @@ CLASS net/minecraft/data/models/BlockModelGenerators
 	CLASS WoodProvider
 		METHOD <init> (Lnet/minecraft/data/models/BlockModelGenerators;Lnet/minecraft/data/models/model/TextureMapping;)V
 			ARG 2 logMapping
+		METHOD <init> (Lnet/minecraft/data/models/model/TextureMapping;)V
+			ARG 1 textureMapping
```

Comment:
Hmm, this looks like an edge case with inner classes (non-`static` nested classes), as the constructor of this method has two arguments, as can be seen in the correct mapping above (the enclosing class instance, and then the regular mandated parameters). @SizableShrimp, could you please take a look at this?

In the meantime, please remove this and the other added mapping in this class.

Reply:
This issue is now being tracked at https://github.com/ParchmentMC/Scribe/issues/5.